src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -96,7 +96,7 @@ export class LcViewer extends PureComponent {
         const LcPeriodInstance=  get(getAppOptions(), 'charts.chartEngine')==='plotly' ? LcPeriodPlotly : LcPeriod;
 
         var mainView = (err,converterId) => {
-            if (!isEmpty(error) && converterId) {
+            if (error!='undefined' && !isEmpty(error) && converterId) {
 
```

Comment:
isEmpty(undefined) will return true.  but here, it's checking for a string 'undefined'.